### PR TITLE
Add old member badges

### PR DESCRIPTION
**Pull request by odinuge**
## Add badges

screenshot from 2018-10-18 21-33-48

Depends on https://github.com/webkom/lego/pull/1295

### CODE REVIEW

**Comment by odinuge** [comment]
screenshot from 2018-10-18 21-33-48

File: app/routes/users/components/UserProfile.js [hunk 1]
```diff
@@ -131,12 +167,36 @@ export default class UserProfile extends Component<Props, UpcomingEventsProps> {
...
+      })),
+      membership =>
+        membership.abakusGroup.logo
+          ? 'membershipsAsBadges'
+          : 'membershipsAsPills'
     );
+
+    const {
+      pastMembershipsAsBadges = []
+      // pastMembershipsAsPills = []
```
Comment (SmithPeder) [inline app/routes/users/components/UserProfile.js]: Comment?

File: app/routes/users/components/UserProfile.js [hunk 2]
```diff
@@ -47,20 +49,52 @@ type UpcomingEventsProps = {
...
+  end
+}: {
+  group: Group,
+  start: Dateish,
+  end: ?Dateish
+}) => (
+  <div>
     {group.name}
-  </Pill>
+    {' ('}
```
Comment (SmithPeder) [inline app/routes/users/components/UserProfile.js]: This looks kinda funky, no other way to format this code?

File: app/routes/users/components/UserProfile.js (hunk 1 above)
Comment (odinuge) [reply app/routes/users/components/UserProfile.js]: Just to make it easy for others to find the code, and understand that `pastMembershipsAsPills` is a thing.

Comment (odinuge) [reply app/routes/users/components/UserProfile.js]: Guess we can remove it. Looks kinda strange

File: app/routes/users/components/UserProfile.js (hunk 2 above)
Comment (odinuge) [reply app/routes/users/components/UserProfile.js]: :+1: 
10/10 unreadable code

File: app/routes/users/components/UserProfile.js [hunk 3]
```diff
@@ -47,20 +49,49 @@ type UpcomingEventsProps = {
...
+  start,
+  end
+}: {
+  group: Group,
+  start: Dateish,
+  end: ?Dateish
+}) => {
+  const startYear = moment(start).year();
+  const endYear = end ? moment(end).year() : 'd.d.';
+  return `${group.name} (${startYear} - ${endYear})`;
```
Comment (odinuge) [inline app/routes/users/components/UserProfile.js]: @SmithPeder This looks a bit better :laughing:

Comment (SmithPeder) [reply app/routes/users/components/UserProfile.js]: That looks a lot better 🥇

**Comment by SmithPeder** [comment]
Was about to merge, but I see it depends on https://github.com/webkom/lego/pull/1295
... and as costom you forgot to say that

**Comment by odinuge** [comment]
Ops. Good catch. Yeah, did it again :upside_down_face: 

Do you think webkom/lego#1295 looks good :thinking: ?

**Comment by SmithPeder** [comment]
LBTM 💯  `Looks Backend To Me`

**Pull request merged**